Src/BootCamp.Chapter/ContactsCenter.cs

```diff
@@ -10,7 +10,34 @@ public class ContactsCenter
 
         public ContactsCenter(string peopleFile)
         {
-            // load people
+            _people = new List<Person>();
+            if (string.IsNullOrWhiteSpace(peopleFile) || !File.Exists(peopleFile))
+            {
+                throw new FileNotFoundException($"The {nameof(peopleFile)} you gave does not exist.");
+            }
+            else
+            {
+                LoadPeople(peopleFile);
+            }
+        }
+
+        private void LoadPeople(string peopleFile)
+        {
+            string[] peoples = File.ReadAllText(peopleFile).Split(Environment.NewLine);
+
+            if (peoples.Length == 1)
+            {
+                throw new Exception($"{peopleFile} was Empty");
+            }
+
+            foreach (string personLine in peoples)
```

Comment:
LIne is redundant, can simply be person in people

Reply:
But its not a person yet and I have an out called person, that is an actual person?